vita3k/shader/src/spirv_recompiler.cpp

```diff
@@ -1528,7 +1528,7 @@ std::string convert_gxp_to_glsl(const SceGxmProgram &program, const std::string
 
 void convert_gxp_to_glsl_from_filepath(const std::string &shader_filepath) {
     const fs::path shader_filepath_str{ shader_filepath };
-    fs::ifstream gxp_stream(shader_filepath_str, fs::ifstream::binary);
+    std::ifstream gxp_stream(shader_filepath_str.filename().string(), std::ifstream::binary);
```

Comment:
no sur have sense here, can directly using shader_filepath if you no using fs::path att all 🤷‍♂️

Reply:
Yes, I also noticed that but for some reason, I thought you might like this way better!
I'll change it to directly pass `shader_filepath`.
`fs` might be `boost::filesystem`, but since we already have `ifstream` in `std::`I think it's better to use it so one place less that uses `boost` :)